sphinxcontrib_autodoc_filterparams/__init__.py

```diff
@@ -34,14 +34,14 @@ def process_docstring(app, what, name, obj, options, lines):
 
 def process_signature(app, what, name, obj, options,
                       signature, return_annotation):
-    filter_params = getattr(app.config, _ext, None)
+    filter_params = ((getattr(app.config, _ext, None)()),)
     stars = bool(getattr(app.config, _ext + '_stars', True))
 
-    if filter_params is None or not callable(obj) or signature is None:
-        return signature
-    if not callable(filter_params):
-        _log.error(_ext + ' is not callable')
-        return signature
+    if enumerate(filter_params) is None or not callable(obj) or signature is None:
```

Comment:
You call `enumerate` on the single-element tuple, which will never be `None` -- we want to check here that the `obj` we are documenting is a callable (function, method, ...).